api/v3/CiviOffice/Convert.php

```diff
@@ -29,37 +29,37 @@ function _civicrm_api3_civi_office_convert_spec(&$spec)
         'name'         => 'document_uri',
         'api.required' => 1,
         'type' => CRM_Utils_Type::T_STRING,
-        'title'        => E::ts('Document URI. E.g.: "local::common/vorlage_kontakte_und_zuwendungen.docx"'),
+        'title'        => E::ts('Document URI, e.g. "local::common/template_contacts_and_contributions.docx"'),
```

Comment:
I find the example file name confusing, let's just use `local::common/example.docx`. Also, the example should not be part of the title, but the description.

Reply:
Changed.